### ISSUE TITLE
build fails on Strawberry perl 5.25.6

### ISSUE THREAD

**Issue by chorny** [tokuhirom/Perl-Lexer#6]
```
In file included from lib\Perl\gen/token_info-latest.h:1:0,
                 from lib\Perl\gen/token_info_map.h:232,
                 from lib\\Perl\\Lexer.xs:16:
lib\Perl\gen/perly-latest.h:56:6: error: 'WORD' redeclared as different kind of symbol
      WORD = 264,
      ^
In file included from C:/Strawberry256/c/i686-w64-mingw32/include/windef.h:8:0,
                 from C:/Strawberry256/c/i686-w64-mingw32/include/windows.h:69,
                 from C:\STRAWB~1\perl\lib\CORE/win32.h:131,
                 from C:\STRAWB~1\perl\lib\CORE/win32thread.h:4,
                 from C:\STRAWB~1\perl\lib\CORE/perl.h:3181,
                 from lib\\Perl\\Lexer.xs:7:
C:/Strawberry256/c/i686-w64-mingw32/include/minwindef.h:140:26: note: previous declaration of 'WORD' was here
   typedef unsigned short WORD;
                          ^
```

**Comment by syohex** [comment]
Token name `WORD` was renamed to `BAREWORD` since Perl 5.25.3. I think this issue is fixed by generating header files for such perl versions(#7).

**Comment by charsbar** [comment]
Shipped 0.20 with the above fix. Thanks.

**Comment by eserte** [comment]
Compile works now, but t/01_simple.t fails with 5.25.3+:
```
        #   Failed test at t/01_simple.t line 29.
        #          got: 'BAREWORD'
        #     expected: 'WORD'
        # Looks like you failed 1 test of 4.

    #   Failed test 'second'
    #   at t/01_simple.t line 32.

        #   Failed test at t/01_simple.t line 36.
        #          got: 'BAREWORD'
        #     expected: 'WORD'
        # Looks like you failed 1 test of 4.

    #   Failed test 'third'
    #   at t/01_simple.t line 39.
    # Looks like you failed 2 tests of 4.

#   Failed test 'use Foo 0.01'
#   at t/01_simple.t line 40.
# Looks like you failed 1 test of 3.
t/01_simple.t ... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/3 subtests 
```

**Comment by charsbar** [comment]
@eserte Thanks! Confirmed and shipped 0.21.

**Comment by eserte** [comment]
Yes, looks good now.
